AK/Userspace.h

```diff
@@ -27,10 +27,14 @@
 #pragma once
 
 #include <AK/Types.h>
+#include <type_traits>
```

Comment:
Please don't use STL headers. Instead, use `IsPointer` from AK/StdLibExtras.h :)

Reply:
Oh, ok. I didn't know that existed. I will do it now.